README.md

```diff
@@ -1,16 +1,21 @@
-# fairscale
+![FairScale Logo](./docs/source/_static/img/fairscale-logo.png)
+
 ![PyPI](https://img.shields.io/pypi/v/fairscale)
 [![Documentation Status](https://readthedocs.org/projects/fairscale/badge/?version=latest)](https://fairscale.readthedocs.io/en/latest/?badge=latest)
 [![CircleCI](https://circleci.com/gh/facebookresearch/fairscale.svg?style=shield)](https://app.circleci.com/pipelines/github/facebookresearch/fairscale/) ![PyPI - License](https://img.shields.io/pypi/l/fairscale) [![PRs Welcome](https://img.shields.io/badge/PRs-welcome-brightgreen.svg)](https://github.com/facebookresearch/fairscale/blob/master/CONTRIBUTING.md)
+--------------------------------------------------------------------------------
 
 ## Description
 fairscale is a PyTorch extension library for high performance and large scale training for optimizing training on one or across multiple machines/nodes. This library extend basic pytorch capabilities while adding new experimental ones.
 
 fairscale supports:
 * Parallelism:
    * pipeline parallelism (fairscale.nn.Pipe)
-   * tensor parallelism (fairscale.nn.model_parallel)
-* Optimization:
+* Sharded training:
+   * Optimizer state sharding
+   * Sharded grad scaler - automatic mixed precision
+   * Sharded distributed data parallel
+* Optimization at scale:
    * optimizer state sharding (fairscale.optim.oss)
```

Comment:
this line becomes a duplicate with the Sharded training / optimizer just above

Reply:
removed... thanks!